test/functional/core/startup_spec.lua

```diff
@@ -576,6 +577,26 @@ describe('user config init', function()
     eq(funcs.fnamemodify(init_lua_path, ':p'), eval('$MYVIMRC'))
   end)
 
+
+  describe 'with existing .exrc in cwd'(function()
+    local exrc_path = '.exrc'
+    before_each(function()
+      write_file(exrc_path, [[
+        let g:from_exrc = 1
+        ]])
+    end)
+
+    after_each(function()
+      os.remove(exrc_path)
+    end)
+
+    it('loads .exrc', function()
+      clear{ args_rm={'-u'}, env={ XDG_CONFIG_HOME=xconfig }}
+      feed('a') -- accept secure read prompt
```

Comment:
This is not working, unfortunately. Does anyone know how could I accept the prompt?
I noticed that there's no exrc test at all, which might be for a reason.

Reply:
You need to attach a UI, otherwise the prompt is skipped.

Reply:
Hmm, exrc isn't enabled here, and it seems a bit tricky to enable it.

Reply:
It should be since it's using `init_lua_path`, which has `vim.opt.exrc = true` set. Unless I'm missing something.

Reply:
Oh wait, I think here the prompt shows up and is skipped before the UI is attached. I guess the `.exrc` cannot be in the directory where `nvim` is startup in this test then. `:cd` into another directory with `.exrc` may work, but I cannot seem to get it to work either.

Reply:
Oh wait, maybe the prompt is skipped before the UI is attached. This is a bit tricky. I guess this test needs to run `nvim` in `:terminal` instead?

Reply:
`:terminal` worked out, thanks!